test/h265_sei_parser_unittest.cc

```diff
@@ -0,0 +1,45 @@
+/*
+ *  Copyright (c) Facebook, Inc. and its affiliates.
+ */
+
+#include "h265_sei_parser.h"
+
+#include <gmock/gmock.h>
+#include <gtest/gtest.h>
+
+#include "h265_common.h"
+#include "rtc_base/arraysize.h"
+#include "rtc_base/bit_buffer.h"
+
+namespace h265nal {
+
+class H265SeiParserTest : public ::testing::Test {
+ public:
+  H265SeiParserTest() {}
+  ~H265SeiParserTest() override {}
+};
+
+TEST_F(H265SeiParserTest, TestUserDataRegisteredItuTT35Sei) {
+  // fuzzer::conv: data
+  const uint8_t buffer[] = {
+    0x04, 0x47, 0xB5, 0x00, 0x31, 0x47, 0x41, 0x39, 0x34, 0x03, 0x54, 0x00,
```

Comment:
we're using non-caps hex (i.e. "0xb5" instead of "0xB5")

Reply:
Ok